test/integrationtests/http_client_helper.go

```diff
@@ -0,0 +1,32 @@
+package integrationtests
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+var defaultClient = &http.Client{}
+
+func sendRequestToLab(url string, query map[string]string, accessToken string) ([]byte, error) {
```

Comment:
this name isn't quite right. The implementation isn't about sending a request to the lab. It is creating an arbitrary request and returning the response body.

Reply:
Tried changing the name to reflect what it does, let me know your thoughts and suggestions if this doesn't seem appropriate